py/torch_tensorrt/dynamo/backend/conversion.py

```diff
@@ -27,11 +27,21 @@ def convert_module(
     Returns:
         TRTModule or TRTModuleNext
     """
+    # Specify module output data types to ensure TRT output types agree with
+    # that of the equivalent Torch module
+    module_outputs = module(*inputs)
```

Comment:
Same thing with this, understand its like 5 lines but its worth having a PR for this as well